mlir/tools/miopen-gen/miopen-gen.cpp

```diff
@@ -1993,8 +2028,7 @@ int main(int argc, char **argv) {
 
   if (!hasUserKernel) {
     if (genCPUKernel.getValue()) {
-      auto func = createCPUConvFunc(module, genConfig);
```

Comment:
We don't want to GPU-wrap a CPU function.  Technically it'll be a kernel, but "kernel" conflates both "computational kernel" and "function run on GPU."